### DISCUSSION TITLE
Plugin stopped loading, Not fully sure why. - Foundry VTT version compatibility

### DISCUSSION THREAD

**Discussion by Kontrano** [misthero/dnd5e-spellpoints#61] [Q&A] [answered]
Hi I seem to have an issue with the module no longer showing up ingame.

I tried installing the V10 version from here but it also won't install and show up.

For reference, I'm on Foundry V10.291 with DnD5e 2.1.5

Is there something I'm missing? or should i update foundry or the DnD module?

With kind regards,

Kontrano

**Comment by misthero** [comment]
which version of the module dnd5e-spellpoints  you have?

**Comment by misthero** [comment, marked as answer]
The latest version compatible with foundry vtt v10 is 1.5.1 and it requires minimum dnd5e system version: 2.2.0

https://github.com/misthero/dnd5e-spellpoints/releases/tag/v1.5.1

if you wanna stay on dnd5e system 2.1.5 you should try version 1.5.0 or previous versions:

https://github.com/misthero/dnd5e-spellpoints/releases

**Comment by Kontrano** [comment]
Thanks for your reply. Sorry, i missed the notification, I ended up upgrading to dnd5e version 2.2 and then getting the 1.5.1 version that indeed worked :)
